crates/sparrow-expressions/src/evaluators.rs

```diff
@@ -88,6 +91,10 @@ impl<'a> StaticArg<'a> {
     pub fn struct_(&self) -> error_stack::Result<StructValue, Error> {
         StructValue::try_new(self.index, self.data_type)
     }
+
+    pub fn _map_(&self) -> error_stack::Result<MapValue, Error> {
```

Comment:
I don't think we need the `_` suffix, since I don't believe that `map` is a keyword in rust. I believe we can just omit this part of the change if we want for now and add it later.